js/storage.js

```diff
@@ -1532,10 +1532,16 @@ Storage.nwLoadTeamFile = function (filename, localApp) {
 		return;
 	}
 
-	var format = '';
+	var format = 'gen8';
+	var capacity = 6;
 	var bracketIndex = line.indexOf(']');
 	if (bracketIndex >= 0) {
 		format = line.slice(1, bracketIndex);
+		if (format && format.slice(0, 3) !== 'gen') format = 'gen6' + format;
+		if (format && format.endsWith('-box')) {
+			format = format.slice(0, -4);
+			capacity = 24;
+		}
 		line = $.trim(line.slice(bracketIndex + 1));
 	}
 	if (format && format.slice(0, 3) !== 'gen') format = 'gen6' + format;
```

Comment:
Even weirder is this line which appears to have no effect (but obviously your copy is significant).

Reply:
gen 6 formats didn't have a gen prefix in gen 6, so somebody with _really_ old teams might have some that this line hasn't fixed yet.

Reply:
Oh, I was confused by the patch. The problem with this line is actually that it's obsoleted by your copy, since there's no other way to have a format by this point.

Reply:
Oh, I see.

Reply:
```suggestion
```